### DISCUSSION TITLE
Are AOA colors accurate for the tomcat?

### DISCUSSION THREAD

**Discussion by jleslie** [DCS-gRPC/lso#8] [Q&A] [answered]
Are the AOA colors drawn in the image accurate for the tomcat?  Or are they based off of the super carrier data which is currently only calibrated for the Hornet?

**Comment by jleslie** [comment]
To answer my own question, it uses a set of data that looks to be calibrated for the hornet.  If pull requests are accepted I may be able to add data for the tomcat as well.

**Comment by rkusa** [comment]
I personally don't fly the Tomcat, so I cannot vouch for the data being correct. The only aircraft specific data is the hook position and glide slope. I am confident that the hook position is correct, but maybe the glide slope is different - I am pretty sure that I just copied it over from the Hornet. You can find the data here: https://github.com/DCS-gRPC/lso/blob/main/src/data.rs#L146-L152
I am accepting PRs.

**Reply by rkusa** [reply, marked as answer]
But I guess this is what needs to be different for the Tomcat (not handled as being aircraft specific right now)? https://github.com/DCS-gRPC/lso/blob/main/src/draw.rs#L355-L373

**Reply by jleslie** [reply]
The AoA for the tomcat and T-45 are different than the F-18.  For the tomcat, an on speed approach will always read as slow currently since the tomcat lands at 15 units.  I have a fork with the work done, it compiled (with sources cited), I just need to do a flight to test it.

**Reply by jleslie** [reply]
PR up with implementation for the F-14 after testing to verify functionality. 

https://github.com/DCS-gRPC/lso/pull/10
